app/src/generate.py

```diff
@@ -20,6 +21,12 @@ def get_models() -> dict[str, str]:
 
     if "OPENAI_API_KEY" in os.environ:
         return {"OpenAI GPT-4o": "gpt-4o"}
+    if "OLLAMA_HOST" in os.environ:
+        ollama_models = {}
+        models = ollama.list()
+        for model in models["models"]:
+            ollama_models[f"ollama/{model['name']}"] = model["name"]
```

Comment:
The key should be the human-readable name and the value should be what LiteLLM expects, so I think this should be:
```suggestion
            ollama_models[f"Ollama {model['name']}"] = f"ollama/{model['name']}"
```